.github/workflows/maven-build.yml

```diff
@@ -69,3 +75,18 @@ jobs:
       
     - name: Build with Maven
       run: mvn ${{ env.debugFlag }} --batch-mode  --file pom.xml clean ${{ inputs.mavenPhases }}
+
+    - name: Upload Checkstyle reports to GitHub
+      uses: jwgmeligmeyling/checkstyle-github-action@v1.2
```

Comment:
This action adds comments in the PR, even for lines that haven't been modified in the PR. When a project is riddled with bad style, etc. (like Jawk), it pollutes the PR with thousands of comments (and it's very long to load).

Can we configure this action to limit comments to parts of the code that are modified in the PR?

Or introduce variables to disable these reports.

Reply:
Do you want a single variable for all reports or 3 variables (1 per report)?